### ISSUE TITLE
Native Image component includes outdated accessibility hint

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#45124]
## Description

When using a screen reader to navigate images in the native editor, an erroneous "Double tap and hold to edit" hint is read aloud. This discrepancy likely originates from https://github.com/WordPress/gutenberg/pull/40651. 

## Step-by-step reproduction instructions

1. Add an Image or Media & Text block. 
2. Add media to the block. 
3. Enable Voice Over. 
4. Move focus to the image. 

## Expected behaviour

An accurate hint is read aloud, e.g. "Double tap to view larger." 

## Actual behaviour

An inaccurate hint is read aloud: "Double tap and hold to edit." 

## Screenshots or screen recording (optional)

https://user-images.githubusercontent.com/438664/196736195-90c75082-05e2-43dc-bdb4-19adf72e9b76.MP4

## WordPress information
- WordPress version:  6.1
- Gutenberg version:  14.3.1
- Are all plugins except Gutenberg deactivated?  Yes
- Are you using a default theme (e.g. Twenty Twenty-One)?  Geologist

## Device information
- Device:  iPhone SE
- Operating system:  iOS 16
- WordPress app version:  1.84.0

**Comment by dcalhoun** [comment]
I explored fixing this briefly, but paused as it was not completely clear if the label should be removed or updated. The a11y hint/state/action also appears to spread across the Image component and Image/Media & Text edit components. Both the Image and Media & Text edit components support various actions depending upon the state of the media, so we should likely have a dynamic accessibility hint. 

https://github.com/WordPress/gutenberg/blob/88a1f7ae18bea3a13f6d1e507e3c25090b76cb08/packages/block-library/src/image/edit.native.js#L331-L346

https://github.com/WordPress/gutenberg/blob/88a1f7ae18bea3a13f6d1e507e3c25090b76cb08/packages/block-library/src/media-text/media-container.native.js#L88-L107

**Comment by fluiddot** [comment]
> I explored fixing this briefly, but paused as it was not completely clear if the label should be removed or updated. The a11y hint/state/action also appears to spread across the Image component and Image/Media & Text edit components. Both the Image and Media & Text edit components support various actions depending upon the state of the media, so we should likely have a dynamic accessibility hint.

Good point @dcalhoun , it's true that the Image component depending on its state should provide different actions. Hence, I also lean towards implementing a dynamic accessibility hint depending on that. However, I'm wondering if we could do this in two steps:
1. Update the hint to `Double tap to view larger.` to avoid confusion with the `hold to edit` message.
2. Introduce dynamic accessibility hints that consider the media states.

WDYT?

**Comment by dcalhoun** [comment]
@fluiddot yes, I agree it likely makes sense to fix the erroneous a11y first, then follow up with more robust, dynamic a11y labels.

**Comment by yashjawale** [comment]
Greetings!
I've added a PR for updating hint as per step 1
Will be happy to make the hint dynamic for step 2

I'm contributing in Gutenberg for the first time so feel free to point out any mistakes or tips for improvement
Thanks!

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
